src/config.cpp

```diff
@@ -353,15 +375,18 @@ namespace config {
       (int) amd::quality_h264_e::balanced,  // quality (h264)
       (int) amd::quality_hevc_e::balanced,  // quality (hevc)
       (int) amd::quality_av1_e::balanced,  // quality (av1)
-      (int) amd::rc_h264_e::cbr,  // rate control (h264)
-      (int) amd::rc_hevc_e::cbr,  // rate control (hevc)
-      (int) amd::rc_av1_e::cbr,  // rate control (av1)
+      (int) amd::rc_h264_e::_auto,  // rate control (h264)
+      (int) amd::rc_hevc_e::_auto,  // rate control (hevc)
+      (int) amd::rc_av1_e::_auto,  // rate control (av1)
+      (int) amd::rc_h264_e::vbr_latency,  // fallback rate control (h264)
+      (int) amd::rc_hevc_e::vbr_latency,  // fallback rate control (hevc)
+      (int) amd::rc_av1_e::vbr_latency,  // fallback rate control (av1)
       (int) amd::usage_h264_e::ultralowlatency,  // usage (h264)
       (int) amd::usage_hevc_e::ultralowlatency,  // usage (hevc)
       (int) amd::usage_av1_e::ultralowlatency,  // usage (av1)
```

Comment:
I will move these lines (and their pairs in `config.h` ) to the top so they match the web UI order in the next refresh.